Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -106,6 +107,9 @@ PrimaryVertexValidation::PrimaryVertexValidation(const edm::ParameterSet& iConfi
     theTrackClusterizer_ = new GapClusterizerInZ(iConfig.getParameter<edm::ParameterSet>("TkClusParameters").getParameter<edm::ParameterSet>("TkGapClusParameters"));
   }else if(clusteringAlgorithm=="DA"){
     theTrackClusterizer_ = new DAClusterizerInZ(iConfig.getParameter<edm::ParameterSet>("TkClusParameters").getParameter<edm::ParameterSet>("TkDAClusParameters"));
+    // provide the vectorized version of the clusterizer, if supported by the build
+  } else if(clusteringAlgorithm=="DA_vect") {
```

Comment:
Hello @arunhep, I am not sure I understand your comment. 
Here I am proposing to move from a legacy and soon-to-become-unsupported algorithm to a more modern one. Depending on who's merged first, either this PR or #18330 might need to rebase. I am fine to wait until the clusterizer tuning is merged and to update the configuration here. 
As for the removal of the old algorithm, I think this will need to be done consistently for all interfaces, so I would let it do it to whomever will take care of that. 
Do you have an alternative proposal?